repo/packages/D/datastax-dse/400/config.json

```diff
@@ -0,0 +1,2196 @@
+          "default": 1024
+        },
+        "authenticator": {
+          "id": "http://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/configCassandra_yaml.html#configCassandra_yaml__authenticator",
+          "type": "string",
+          "description": "The authentication backend. It implements IAuthenticator, which is used to identify users.",
+          "default": "AllowAllAuthenticator"
+        },
+        "authorizer": {
+          "id": "http://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/configCassandra_yaml.html#configCassandra_yaml__authorizer",
+          "type": "string",
+          "description": "The authorization backend. It implements IAuthenticator, which limits access and provides permissions.",
+          "default": "AllowAllAuthorizer"
+        },
+        "role_manager": {
+          "type": "string",
+          "default": "com.datastax.bdp.cassandra.auth.DseRoleManager"
+        },
+        "roles_validity_in_ms": {
+          "type": "integer",
+          "default": 2000
+        },
+        "roles_update_interval_in_ms": {
+          "type": "integer",
+          "default": 2000
+        },
+        "permissions_validity_in_ms": {
+          "type": "integer",
+          "default": 2000
+        },
+        "permissions_update_interval_in_ms": {
+          "type": "integer",
+          "default": 1000
+        },
+        "credentials_validity_in_ms": {
+          "type": "integer",
+          "default": 2000
+        },
+        "credentials_update_interval_in_ms": {
+          "type": "integer",
+          "default": 2000
+        },
+        "partitioner": {
+          "id": "http://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/configCassandra_yaml.html#configCassandra_yaml__partitioner",
+          "type": "string",
+          "description": "The partitioner used to distribute rows across the cluster. Murmur3Partitioner is the recommended setting. RandomPartitioner and ByteOrderedPartitioner are supported for legacy applications.",
+          "default": "org.apache.cassandra.dht.Murmur3Partitioner"
+        },
+        "cdc_enabled": {
+          "type": "boolean",
+          "default": false
+        },
+        "disk_failure_policy": {
+          "type": "string",
+          "default": "stop"
+        },
+        "commit_failure_policy": {
+          "type": "string",
+          "default": "stop"
+        },
+        "prepared_statements_cache_size_mb": {
+          "type": "string",
+          "default": ""
+        },
+        "thrift_prepared_statements_cache_size_mb": {
+          "type": "string",
+          "default": ""
+        },
+        "key_cache_size_in_mb": {
+          "type": "string",
+          "default": ""
+        },
+        "key_cache_save_period": {
+          "id": "hhttp://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/configCassandra_yaml.html#configCassandra_yaml__key_cache_save_period",
```

Comment:
Typo: `hhttp`.

Reply:
Also, for my own understanding, is the `id` field used by any of our tooling, or is it mostly metadata for people looking at these configuration files?

Reply:
I will ping the DSE channel regarding this.

I don't think we explicitly use ID in our tooling, but it could be that the UI renders the ID field in some way.